MockKitUI/View Controllers/MockCGMManagerSettingsViewController.swift

```diff
@@ -15,13 +16,28 @@ import MockKit
 final class MockCGMManagerSettingsViewController: UITableViewController {
     let cgmManager: MockCGMManager
 
-    private var glucoseUnit: HKUnit
+    private var displayGlucoseUnitObservable: DisplayGlucoseUnitObservable
 
-    init(cgmManager: MockCGMManager, glucoseUnit: HKUnit) {
+    private var cancellable: AnyCancellable?
+
+    private var glucoseUnit: HKUnit {
+        displayGlucoseUnitObservable.displayGlucoseUnit
+    }
+
+    init(cgmManager: MockCGMManager, displayGlucoseUnitObservable: DisplayGlucoseUnitObservable) {
         self.cgmManager = cgmManager
-        self.glucoseUnit = glucoseUnit
+        self.displayGlucoseUnitObservable = displayGlucoseUnitObservable
+
         super.init(style: .grouped)
         title = NSLocalizedString("CGM Settings", comment: "Title for CGM simulator settings")
+
+        cancellable = displayGlucoseUnitObservable.updatePublisher.sink { [weak self] in
```

Comment:
[nit] Consider above comment, plus:
```
$displayGlucoseUnitObservable
    .sink { [weak self] in self?.tableView.reloadData() }
    .store(in: &cancellables)
```
Not sure if this will work here.